documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
That didn't work. Instead of using an email attachment, can you drag and drop (or paste) the screenshot into the GitHub webpage?